WcaOnRails/app/models/user.rb

```diff
@@ -401,6 +402,7 @@ def self.delegate_status_requires_senior_delegate(delegate_status)
     {
       nil => false,
       "" => false,
+      "trainee_delegate" => true,
```

Comment:
Again to answer the question you raised in the issue: yes you definitely need that!
The change in the js file performs client-side validation, but it would be very easy for someone to craft a request that bypasses this validation.
Therefore we also need server-side validation, which is ensured by the validation a couple of lines above that function.
(By reading the code I actually realize that we may be doing the validation twice...)

Reply:
Okay got it. Thanks :-)